code/mob/living/carbon/human/procs/Life.dm

```diff
@@ -110,7 +110,7 @@
 
 /mob/living/carbon/human/Life(datum/controller/process/mobs/parent)
 	set invisibility = 0
-	if (..(parent))
+	if (..())
```

Comment:
why remove the parent in the ..() here?

Reply:
imo its not good practice to manually pass args unless you want to make any changes to the values being passed - it happens automatically when blank so it just makes the code more annoying to work with if people add args later

Reply:
`parent` does not need to be specified, it is included automagically in `..()`, it is better to not confuse people when 99.999% of parent calls use the implicit argument form.